lecture_app_engine.py

```diff
@@ -0,0 +1,87 @@
+import json
+import os
+import asyncio
+import sys
+
+from dotenv import dotenv_values
+
+import openai_client
+from lecture_parser import lecture_factory
+
+WINDOWS_PLATFORM = 'win'
+
+
+class AppEngine:
+    ENV_FILE_PATH = ".env"
+    OPENAI_API_KEY = "OPENAI_API_KEY"
```

Comment:
you can put it in the environment variables

Reply:
I preferred to use a .env file because it allows you to keep all your configuration variables, including API keys, in one central location. This makes it easier to manage and distribute your code, as you can simply include the .env file along with the application. On the other hand, using environment variables requires setting them up individually on each system, which can be cumbersome and error-prone.